omod/src/main/webapp/resources/scripts/filters/display.js

```diff
@@ -6,7 +6,7 @@ angular.module('uicommons.filters').
      * If no translation is found, just return input.
      * (If input is an array, these rules will be applied to each element, and they'll be joined with ", ".)
      */
-    filter('omrs.display', function() {
```

Comment:
Was this also part of the angular upgrade? Do you mind explaining the reason behind it?